contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -22,10 +28,8 @@ mod SequencerUptimeFeed {
     use chainlink::ocr2::aggregator::Aggregator::TransmissionStorageAccess;
     use chainlink::libraries::upgradeable::Upgradeable;
 
-    const ETH_ADDRESS_BOUND: felt252 = 0x10000000000000000000000000000000000000000; // 2**160
-
     struct Storage {
-        // l1 sender is an ethereum address
+        // l1 sender is an starknet validator ethereum address
         _l1_sender: felt252,
```

Comment:
Are we not able to use `EthAddress` here? I'm guessing it doesn't implement storage?

Reply:
It would probably result in the same bytecode: a cast followed by a store